src/elements/common/modal.scss

```diff
@@ -6,6 +8,10 @@
         justify-content: center;
         padding: 15px 0 0;
 
+        [class^='bp_base_button_module_button']:last-child {
+            margin-inline-start: tokens.$space-3;
```

Comment:
i think now we should move towards using the css vars:

```
@use '@box/blueprint-web-assets/tokens/tokens-css-vars.scss';
...

.component {
    background-color: var(--background-background);
}
```

the scss tokens are deprecated in the docs